Lib/test/test_asyncio/test_events.py

```diff
@@ -2701,37 +2701,6 @@ def test_event_loop_policy(self):
         self.assertRaises(NotImplementedError, policy.set_event_loop, object())
         self.assertRaises(NotImplementedError, policy.new_event_loop)
 
-    def test_get_event_loop(self):
```

Comment:
I do not think that we can just remove this test. `policy.get_event_loop()` still needs to be tested, but with different expected result.

Reply:
fixed